querybook/webapp/components/QueryComposer/QueryComposer.tsx

```diff
@@ -497,62 +497,80 @@ const QueryComposer: React.FC = () => {
         return getSelectedQuery(query, selectedRange);
     }, [query, queryEditorRef]);
 
+    const getSamplingTables = useCallback(
```

Comment:
@jczhong84 I added this to run If sampled_table is not provided while sampling is true and we'll use the TABLESAMPLE clause, if that looks fine to you